unfoldingWord/translationCore
# Hide Non-Alignment Tools

#### This pull request addresses:
This PR removes other tools from being shown


#### How to test this pull request:
Open tC and ensure that the only tool able to load is the wordAlignmentTool


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2828)
